sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -228,6 +229,29 @@ def cross_product(elem, sides):
           pcoll | beam.FlatMap(cross_product, beam.pvalue.AsList(derived)),
           equal_to([('a', 'a'), ('a', 'b'), ('b', 'a'), ('b', 'b')]))
 
+  def test_pardo_state_only(self):
```

Comment:
Is there a way we can unify this with the existing state / timer tests in userstate_test.py?  This isn't FnApiRunner specific.

Reply:
I put these here because it runs them in various modes (directly, over Grpc with and without multiple threads, over subprocesses). I view the ones in userstate_test as more user-api-facing ones (though note that those can't be used here, as they use the streaming test stream and avoid assert_that due to it being broken in streaming mode on the other direct runner, and instead have a DoFn that stores things into a list (which of course won't work cross-process). 

So, yes, it'd be nice to unify, but not right now.

(What I'd like to do eventually is expand this suite of tests into a basic Runner Compatibility Matrix test...)